src/components/Button.js

```diff
@@ -1,8 +1,41 @@
-import React from "react";
-import "./Button.css";
+import styled from "@emotion/styled";
 
-function Button(props) {
-  return <button {...props} />;
-}
+const ButtonDisabled = styled.button`
```

Comment:
for this you could use the `:disabled` selector and only change the properties that change if it is disabled, saves some lines of code. DRY Don't repeat yourself :)